components/checkbox/src/test.tsx

```diff
@@ -1,35 +1,29 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import { shallow, mount } from 'enzyme';
+import { render } from '@testing-library/react';
 
 import Checkbox from '.';
 
 describe('Checkbox', () => {
-  const example = 'example';
-  const wrapper = <Checkbox>{example}</Checkbox>;
-
   it('renders without crashing', () => {
-    const div = document.createElement('div');
-    ReactDOM.render(wrapper, div);
+    render(<Checkbox>Example</Checkbox>);
   });
 
   it('should render an input type="checkbox"', () => {
-    const output = shallow(wrapper);
-    expect(output.find('input[type="checkbox"]')).toBeTruthy();
-  });
+    const { getByRole, getByText } = render(<Checkbox>Example</Checkbox>);
 
-  it('should render a label', () => {
```

Comment:
do we not want this test anymore?

Reply:
maybe a test using getByLabel?

Reply:
hmm - I kinda looked on this as essentially being covered by other tests, plus also this being an implementation detail

Reply:
I've changed things around a bit so this is now explicitly covered

Reply:
thanks, being a label isn't an implementation detail - to users, it's an accessibility feature, using react testing library I guess this is byLabelText https://testing-library.com/docs/queries/bylabeltext/

Reply:
yup - revised test uses `getByLabelText`